lib/ansible/modules/apt.py

```diff
@@ -75,7 +75,7 @@
     type: bool
   force:
     description:
-      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies C(allow_unauthenticated: yes)'
+      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies C(allow_unauthenticated: yes) and C(allow_downgrade: yes)'
```

Comment:
Wouldn't this also mean it's a breaking change, since existing uses of `force: yes` would have different behavior? If so, it needs to be labeled as such in the changelog. Also given how widely this module is used that might not be a palatable change.

Reply:
The commit does not change the way `force: yes` works. I just updated the documentation to reflect the fact that `force: yes` also implies allowing downgrades which is already the case, but now I'm adding an explicit `allow_downgrade` it should be reflected in the docs too.